certif/tests/acceptance/session-details-certification-candidates-test.js

```diff
@@ -180,64 +177,58 @@ module('Acceptance | Session Details Certification Candidates', function (hooks)
         assert.dom(within(modal).getByText('EXTERNAL-ID')).exists();
       });
 
-      module(
-        'when feature toggle isNeedToAdjustCertificationAccessibilityEnabled is true and center is v3 pilot',
-        function (hooks) {
-          let allowedCertificationCenterAccess;
-          let certificationPointOfContact;
-          let session;
-
-          hooks.beforeEach(async () => {
-            server.create('feature-toggle', {
-              isNeedToAdjustCertificationAccessibilityEnabled: true,
-            });
-            allowedCertificationCenterAccess = server.create('allowed-certification-center-access');
-            certificationPointOfContact = server.create('certification-point-of-contact', {
-              firstName: 'Lena',
-              lastName: 'Rine',
-              allowedCertificationCenterAccesses: [allowedCertificationCenterAccess],
-              pixCertifTermsOfServiceAccepted: true,
-            });
-            session = server.create('session-enrolment', {
-              certificationCenterId: allowedCertificationCenterAccess.id,
-            });
-            server.create('certification-candidate', {
-              firstName: 'John',
-              lastName: 'Doe',
-              sessionId: session.id,
-              accessibilityAdjustedCertificationNeeded: true,
-            });
-            server.create('session-management', {
-              id: session.id,
-            });
-            await authenticateSession(certificationPointOfContact.id);
+      module('when center is v3 pilot', function (hooks) {
```

Comment:
Question: Du coup on doit l'afficher dans tous les cas non ?
Auquel cas, ce module n'a plus de raison d'être.

Reply:
Ah peut-être !
J'ai juste enlevé la mention du FT dans l'ancien intitulé "'when feature toggle isNeedToAdjustCertificationAccessibilityEnabled is true and center is v3 pilot'"